### ISSUE TITLE
Set load_namespaces = True by default?

### ISSUE THREAD

**Issue by rly** [NeurodataWithoutBorders/pynwb#1169]
Should we set load_namespaces = True by default for `NWBHDF5IO`? I don't see why you would not want to load the cached specs that are usually necessary to read the file, but maybe I am not thinking of a use case.

**Comment by t-b** [comment]
The validator should be able to not use the cached spec or?

**Comment by rly** [comment]
Yes, but the validator can say load_namespaces=false explicitly and internally.

**Comment by t-b** [comment]
In that case let's do it.

**Comment by ajtritt** [comment]
One thing to consider... The API is written to support the most current state of the specification. This might not play well with an old version of the spec.
